pandas/conftest.py

```diff
@@ -99,3 +99,17 @@ def compression_no_zip(request):
 def datetime_tz_utc():
     from datetime import timezone
     return timezone.utc
+
+
+JOIN_TYPES = ['inner', 'outer', 'left', 'right']
+
+
+# workaround for parametrized tests
```

Comment:
why is this a work-around?

Reply:
can you give a nice doc-string to this, it shows up when we list fixtures

Reply:
@jreback Sure.
Workaround because I cannot use join_types fixture in parametrize, there is a proposal for this in pytest but still not implemented. I will remove it to avoid confusion :)